src/configs/YITHPlugins.config.js

```diff
@@ -8,6 +8,7 @@ import { ReactComponent as CustomizeAccount } from "../icons/yith-woocommerce-cu
 import { ReactComponent as Gift } from "../icons/yith-woocommerce-gift-card.svg";
 import { ReactComponent as WishList } from "../icons/yith-woocommerce-wishlist.svg";
 import { MarketplaceSdk } from "../sdk/marketplace";
+import { ReactComponent as InstallationUI } from '../icons/yith-woocommerce-installation.svg';
```

Comment:
Can you rename the file. May be to `yith-sales-funnel.svg`

Reply:
done